common/utils/src/main/resources/error/error-conditions.json

```
@@ -5309,6 +5309,11 @@
           "Subquery expressions are not supported within higher-order functions. Please remove all subquery expressions from higher-order functions and then try the query again."
         ]
       },
+      "INLINE_TABLE_CONTAINS_SCALAR_SUBQUERY" : {
+        "message" : [
+          "Scalar subqueries inside VALUES clause are not supported."
```

The parent's error format has already the words `not supported`. See the entire error message:
```
Unsupported subquery expression: Scalar subqueries inside VALUES clause are not supported.
```
Please, leave it as:
```suggestion
          "Scalar subqueries in the VALUES clause."
```